js/functions.js

```diff
@@ -0,0 +1,82 @@
+const validateStringLength = (string, maxLength) => string.length <= maxLength;
+
+function isStringPalindrome(string) {
+  let result = string.toLowerCase()
+    .split(" ")
+    .join("")
+    .split("");
+
+  return result.join("") === result.reverse().join("");
+}
```

Comment:
Для того чтобы убрать пробелы можно использовать такую конструкцию
```suggestion
function isStringPalindrome(string) {
  let result = string.toLowerCase()
    .replaceAll(" ", "")
    .split("");

  return result.join("") === result.reverse().join("");
}
```